jepsen/src/jepsen/control/util.clj

```diff
@@ -78,15 +78,29 @@
 
 (defn wget!
   "Downloads a string URL and returns the filename as a string. Skips if the
-  file already exists."
+  file already exists.
+
+  Options:
+
+    force?      Even if we have this cached, download the tarball again anyway.
+    user        user for wget authentication. If provided, valid pw must also be provided.
+    pw          password for wget authentication."
   ([url]
-   (wget! url false))
-  ([url force?]
-   (let [filename (.getName (file url))]
-     (when force?
-       (exec :rm :-f filename))
+   (wget! url {:force? false}))
+  ([url opts]
+   (let [filename (.getName (file url))
+         wget-opts std-wget-opts]
+     ; second parameter was changed from a boolean flag (force?) to an options map
+     ; this check is here for backwards compatibility
+     (when (map? opts)
```

Comment:
@aphyr I added the `map?` check here. `wget` behaves very similarly to `cached-wget`